temporalio/lib/temporalio/client/connection/workflow_service.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'temporalio/api'
+require 'temporalio/client/connection/service'
+require 'temporalio/internal/bridge/client'
+
+module Temporalio
+  class Client
+    class Connection
+      # Raw gRPC workflow service as defined at
+      # https://github.com/temporalio/api/blob/master/temporal/api/workflowservice/v1/service.proto.
+      class WorkflowService < Service
+        rpc_methods(
```

Comment:
While this kind of runtime method definition is nice, we probably want some kind of code gen for `@!method` Yard tags for every method